Volume/Views/PublicationList.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  PublicationList.swift
+//  Volume
+//
+//  Created by Cameron Russell on 10/29/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import SwiftUI
+
+struct PublicationList: View {
+    var body: some View {
+        NavigationView {
+            ScrollView(showsIndicators: false) {
+                sectionOne
+                sectionTwo
```

Comment:
Up to you, but these names seem super unspecific and a little confusing to me

Reply:
lol, yeah you're right